falcon/request.py

```diff
@@ -636,7 +636,8 @@ def client_prefers(self, media_types):
         Args:
             media_types (iterable of str): One or more Internet media types
                 from which to choose the client's preferred type. This value
-                **must** be an iterable collection of strings.
+                **must** be an iterable collection of strings. The iterable
+                should be sorted in the order of increasing desirability.
```

Comment:
OK, I can do that. And modify the docstring to suit. I suppose this would qualify as a breaking change though. We'll have to hold this one out for the next major version bump.